roles/slurm-client/handlers/main.yml

```diff
@@ -10,14 +10,16 @@
 - name: Restart munge service.
   systemd:
     name: 'munge.service'
-    state: restarted
+    state: 'restarted'
+    daemon_reload: 'yes'
   become: true
   listen: restart_munge
 
 - name: Restart slurmd service.
   systemd:
     name: 'slurmd.service'
-    state: reloaded
+    state: 'restarted'
```

Comment:
I'm pretty sure this  will fix our issues concerning the restart of slurm. As they dissappeared after a daeomon-reload.